game/scripts/vscripts/settings.lua

```diff
@@ -133,7 +133,7 @@ END_GAME_ON_KILLS = false               -- Should the game end after a certain n
 KILLS_TO_END_GAME_FOR_TEAM = 50         -- How many kills for a team should signify an end of game?
 
 USE_CUSTOM_HERO_LEVELS = true           -- Should we allow heroes to have custom levels?
-MAX_LEVEL = 5000                        -- What level should we let heroes get to?
+MAX_LEVEL = 50                        -- What level should we let heroes get to?
```

Comment:
Just to check. Have you tested this? I'm not sure if this value is used or if only the XP per level table matters. Also, do align the comments if you don't mind.

And you should probably modify the XP per level table even if this does work as it would have thousands of unnecessary values otherwise.

Reply:
Yes i tried it and i couldn't lvl up past level 50. 
I will take a look at the table.

Reply:
luckely you implemented the "xp_per_level_table" in such a way, that it only calculates the xp up to the value of "MAX_LEVEL".
So by changing the max_level, the size of the table changes automaticaly.

Reply:
Oh? Dur. Maybe I should actually read code that I suggest changes to.